src/RateLimit.js

```diff
@@ -143,12 +143,11 @@ class RateLimit {
         const key = await this.keyGenerator(ctx);
 
         const { counter, dateEnd } = await this.store.incr(key, this.options);
-        const reset = new Date(dateEnd).getTime();
```

Comment:
Use `new Date(dateEnd)` instead of directly use "dateEnd", in case of the store return a string as a Date

Reply:
Will fix